src/components/Wizard/BlueprintWizard.js

```diff
@@ -87,6 +87,20 @@ const BlueprintWizard = (props) => {
         ? formValues
         : formApi.getState().values;
     const blueprintData = formStateToBlueprint(formState);
+
+    // when customizations are provided, the users and filesystem customizations
+    // are initialized to an empty array. This creates issues for image types that
+    // aren't allowed these customizations
+    if (blueprintData.customizations) {
```

Comment:
Ideally we would make sure that this isn't set in the first place. But the data driven forms state is a bit awkward